### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by haslinghuis**
Updates: 
```
yarn add yarn
yarn add cordova-lib

yarn add @babel/core
yarn add @fortawesome/fontawesome-free
yarn add @rollup/plugin-alias
yarn add @rollup/plugin-commonjs
yarn add @rollup/plugin-node-resolve
yarn add @rollup/plugin-replace
yarn add babel-loader
yarn add chai
yarn add command-exists
yarn add del
yarn add dompurify
yarn add eslint
yarn add follow-redirects
yarn add fs-extra
yarn add glob
yarn add gulp-json-editor
yarn add gulp-replace
yarn add gulp-json-editor
yarn add i18next
yarn add jbox
yarn add karma
yarn add karma-chrome-launcher
yarn add karma-mocha
yarn add karma-rollup-preprocessor
yarn add karma-spec-reporter
yarn add marked
yarn add mocha
yarn add object-hash
yarn add os
yarn add postcss
yarn add rollup
yarn add run-script-os
yarn add semver-min
yarn add short-unique-id
yarn add simple-git
yarn add sinon
yarn add sinon-chai
yarn add temp

yarn add @storybook/addon-actions
yarn add @storybook/addon-essentials
yarn add @storybook/addon-links
yarn add @storybook/vue
```

For complete list: https://github.com/betaflight/betaflight-configurator/issues/2958

### CODE REVIEW

**Comment by haslinghuis** [comment]
Remaining:
```
yarn add husky
yarn add inflection
yarn add three

yarn add @storybook/addon-actions
yarn add @storybook/addon-essentials
yarn add @storybook/addon-links
yarn add @storybook/vue
yarn add rollup-plugin-vue
yarn add eslint-plugin-vue

yarn add vue
yarn add vue-loader
yarn add vue-template-compiler
```

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by McGiverGim — APPROVED** [review]
I did some testing and seems to be working. Is a good moment to merge this, to see if we can catch some bug or problem before the next release.

**Review by blckmn — APPROVED** [review]

**Pull request merged**
